src/components/SearchCollections/SearchCollections.js

```diff
@@ -62,7 +62,10 @@ class SearchCollections extends React.Component {
           <StyledDiv>
             {collectionData.results.map((collection) => {
```

Comment:
I guess above you could be doing just one check and get rid of the other, then just do a ternary and render both depending if true or false

Reply:
If I use a ternary, while the information is loading it will display the message that there is no data for collections (or photos or users depending on which tab you're on) so that won't really work the same or the way I want it to.